tests/integration/netapi/test_client.py

```diff
@@ -341,16 +348,18 @@ def test_shell_inject_ssh_options(self):
             "roster": "cache",
             "client": "ssh",
             "tgt": "127.0.0.1",
-            "renderer": "cheetah",
+            "renderer": "jinja|yaml",
```

Comment:
**question:** what does jinja|yaml do different than cheetah? I know the jinja|yaml part but not sure what is the difference?

Reply:
It renders the roster file properly.

Reply:
i will make sure it still fails without the initial fix and report back

Reply:
confirmed changing the renderer change does not impact anything.